.github/workflows/diff-ceph-config.yml

```diff
@@ -0,0 +1,95 @@
+name: Check if any ceph configuration has been changed
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened]
+    branches:
+      - main
+
+# The following permissions are needed to write a comment to repo
+permissions:
+    issues: write
+    contents: write
+    pull-requests: write
+
+jobs:
+  pull_request:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout ceph.git
+        uses: actions/checkout@v4
+        with:
+          path: ceph
+          sparse-checkout: src/script
+          sparse-checkout-cone-mode: false
+
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.13' 
+
+      - name: Install python packages
+        run: |
+          pip3 install -r ./src/script/config-diff/requirements.txt
+        working-directory: ceph
+
+      - name: execute config diff tool
+        id: diff_tool
+        run:  |
+          rm -f diff_result.json
+          python3 ./src/script/config-diff/config_diff.py diff-branch-remote-repo --ref-repo ${{ github.event.pull_request.base.repo.clone_url }} --ref-branch ${{ github.event.pull_request.base.ref }} --remote-repo ${{ github.event.pull_request.head.repo.clone_url }} --cmp-branch ${{ github.event.pull_request.head.ref }} >> diff_result.json
+        working-directory: ceph
+      
+      - name: Post output as a comment
```

Comment:
What do you think about also tagging the PR as "conf change" (or any other name)? Could help with filtering relevant PRs

Reply:
nit: labeler change could be moved to a separate commit as it's not related to the script/workflow.